wix-bookings/guides/Wix Bookings Life Cycle.md

```diff
@@ -0,0 +1,33 @@
+# Wix Bookings Lifecycle
+
+To manage bookings, a life cycle status is used to track their progression through different stages. This status is essential for controlling the possible actions that can be taken, such as confirming a pending booking. By utilizing this life cycle status, the system effectively regulates the booking process from start to finish.
+
+A booking can go through checkout for payment or have no payment, in any case, the validation occurs through [Confirm or decline](https://dev.wix.com/api/rest/wix-bookings/confirmation/confirm-or-decline-booking), where availability, payment and the need for business confirmation are checked.
+
+A booking does not need payment to be confirmed, but payment will make sure that the booking is confirmed, even in a case of `DOUBLEBOOKING`, another way of allowing `DOUBLEBOOKING` is passing `skipAvailability` as `true` when creating the booking. 
+
+The following table describes all the possible status for a booking:
+
+| Status | Definition |
+|---------------|-------------------------|
+
+| `CREATED` | The booking has been created, but doesn't yet appear in the business calendar. 
+The first step and status of a booking is created, it can be used for checkout or processed without it.
+You can create a booking with [Create booking](https://dev.wix.com/api/rest/wix-bookings/bookings-v2/create-booking)
+| `CONFIRMED` | The site owner has confirmed the booking and it appears in the business calendar.
+    + You can manually confirm a booking from pending with [Confirm Booking](https://dev.wix.com/api/rest/wix-bookings/bookings-v2/confirm-booking).
+    + Bookings are automatically confirmed when the [service](https://dev.wix.com/api/rest/wix-bookings/services/service/create-service) is configured to do so and the eCommerce order (coming soon) has been approved. An automatic confirmation includes checking the slot's or schedule's availability.
+    + Bookings are also automatically confirmed if `skipBusinessConfirmation` is passed as `true` when creating the booking, `false` is default.
+| `PENDING` | The booking is waiting to be confirmed or declined by the owner and is displayed in the business calendar.
+You can't manually set bookings as `PENDING` via an API.
+Bookings are automatically set as `PENDING` when an eCommerce order related to the booking has been created (coming soon).
+| `WAITING_LIST` | The booking is pending on a waiting list.
+You must use the [Waitlist APIs](https://dev.wix.com/api/rest/wix-bookings/waitlist/introduction) 
+to create bookings in status `WAITING_LIST`. You can't change a booking's status from `CREATED` to `WAITING_LIST` with the Bookings V2 APIs.
```

Comment:
"You can't change a booking's status from `CREATED` to `WAITING_LIST` with the Bookings V2 APIs." 

Huh?   Is this a limitation for now?   This should be in the Intro "Before you begin" section also.  As is written in our [guidelines in the cafe](https://wix-private.github.io/tw-cafe/style-guide/api-documentation-guidelines/intro-articles/main-intro-article), step 6: "The section helps us inform app developers and Velo users about missing or unexpected functionality. It's important to do this before they start working. This way they won’t get stuck in the middle of their project."

Reply:
adding to before you begin for BV2